kroxylicious-runtime/src/main/resources/META-INF/services/io.kroxylicious.proxy.clusternetworkaddressconfigprovider.ClusterNetworkAddressConfigProviderContributor

```diff
@@ -1,2 +1,3 @@
 io.kroxylicious.proxy.internal.clusternetworkaddressconfigprovider.PortPerBrokerContributor
```

Comment:
Should we deprecate port per broker?

Reply:
I think I'd be happy to let the new feature bake for a while in the community and defer a decision on deprecation.  
 I see this work as an improvement, but not really a desired end state.